usr.bin/sockstat/sockstat.1

```diff
@@ -46,6 +47,13 @@ domain sockets.
 .Pp
 The following options are available:
 .Bl -tag -width Fl
+.It Fl -libxo
+Generate output via
+.Xr libxo 3
+in a selection of different human and machine readable formats.
+See
+.Xr xo_parse_args 3
```

Comment:
Please link to xo_options.7 instead.  I'm going to make that change globally in https://reviews.freebsd.org/D51354 .